web.md

```diff
@@ -0,0 +1,37 @@
+# Web
+
+**Prerequisite knowledge**: 
+
+## Intro
+Web - short for World Wide Web.
+
+No prerequisite knowledge required. If you see this document, you are probably aware of what computers are and that they can communicate with each other via global network of computer networks, called the Internet. If so, you're all set to begin your journey with the web.
+
+## Classification / acceptance questions
+### Apprentice:
+- I can tell what World Wide Web is.
+- I can tell the difference betweend web and the internet.
+- I know what a web browser is.
+- I know what a web server is.
+
+### Intermediate
+- I can explain how does the Web work.
+- I know what DNS is.
+- I can describe step by step what happens when I put `www.youtube.com` into the address bar and hit enter before I can see the results.
+- I can make the above happen for my own custom domain name, which includes:
+  - regestering a domain name
```

Comment:
in my opinion this should be in advanced section.

Beginner - understands the concept of the web
Intermediate - knows about DNS resolving and understands the basics of communication between browser and server
Advanced - knows how to set up this environment on her own